src/Features/Core/Portable/FindUsages/DefinitionItem.cs

```diff
@@ -154,7 +154,7 @@ internal abstract partial class DefinitionItem
         }
 
         public abstract bool CanNavigateTo(Workspace workspace);
-        public abstract bool TryNavigateTo(Workspace workspace, bool isPreview);
+        public abstract bool TryNavigateTo(Workspace workspace, bool isPreview, bool activatePreview);
```

Comment:
could use a doc on how isPreview differs from activatePreview.  what does isPreview control vs activatePreview

Reply:
activatePreview is only relevant with isPreview right?  in that case, this should likely be an enum with three states representing the different behaviors.

Reply:
Good idea... I'll cahnge to an enum